apps/agreement/test/agreement/agreement_dispute.js

```diff
@@ -126,11 +126,11 @@ contract('Agreement', ([_, someone, submitter, challenger]) => {
                       assertEvent(receipt, 'EvidenceSubmitted', { index: 1, expectedArgs: { arbitrator: disputable.arbitrator, disputeId, submitter: challenger, evidence: challengeContext, finished: true }, decodeForAbi: disputable.abi })
                     })
 
-                    it('allows closing the evidence submission period manually', async () => {
```

Comment:
We should probably still keep this test to manually close it though?

Reply:
We do have some for this case in the evidence test file

Reply:
We do have some for this case in the evidence test file